source/objects/responseClass.m

```diff
@@ -250,6 +250,11 @@
             %         the number of MoorDyn lines
             %
             
+            arguments
+                obj
+                numLines (1,1) double {mustBeInteger}
+            end
```

Comment:
Referring to #822, should this be changed to `linesNum` for consistency with e.g. `bodyNum`